src/modules/users/CreateUserJob.cpp

```diff
@@ -116,12 +116,11 @@ CreateUserJob::exec()
         QString autologinGroup;
         if ( gs->contains( "autologinGroup" ) &&
              !gs->value( "autologinGroup" ).toString().isEmpty() )
-            autologinGroup = gs->value( "autologinGroup" ).toString();
-        else
-            autologinGroup = QStringLiteral( "autologin" );
-
-        CalamaresUtils::System::instance()->targetEnvCall( { "groupadd", autologinGroup } );
-        defaultGroups.append( QString( ",%1" ).arg( autologinGroup ) );
+	{
```

Comment:
This entire block needs to be indented so that it aligns with the `if()`. Please run `ci/calamaresstyle` to stick with the coding conventions.